docker/local/polygon-edge.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+POLYGON_EDGE_BIN=./polygon-edge
+
+case "$1" in
+
+   "init")
+      echo "Generating secrets..."
+      node1id=$("$POLYGON_EDGE_BIN" secrets init --data-dir /data1 | grep Node | awk -F ' ' '{print $4}')
+      node2id=$("$POLYGON_EDGE_BIN" secrets init --data-dir /data2 | grep Node | awk -F ' ' '{print $4}')
+      node3id=$("$POLYGON_EDGE_BIN" secrets init --data-dir /data3 | grep Node | awk -F ' ' '{print $4}')
+      node4id=$("$POLYGON_EDGE_BIN" secrets init --data-dir /data4 | grep Node | awk -F ' ' '{print $4}')
+      echo "Secrets have been successfully generated"
+
+      echo "Generating genesis file..."
+      "$POLYGON_EDGE_BIN" genesis --dir /genesis/genesis.json --consensus ibft --ibft-validators-prefix-path test-chain- --bootnode /ip4/127.0.0.1/tcp/10001/p2p/"$node1id" --bootnode /ip4/127.0.0.1/tcp/20001/p2p/"$node2id"
```

Comment:
Shouldn't this `--ibft-validators-prefix-path test-chain-` be `--ibft-validators-prefix-path data`?

Reply:
does not really matter, changed to data